web/gds-user-ui/src/modules/dashboard/member/memberType.ts

```diff
@@ -14,5 +14,6 @@ export enum DirectoryTypeEnum {
 }
 export enum VaspDirectoryEnum {
   TESTNET = 'testnet.net',
-  MAINNET = 'vaspdirectory.net'
+  MAINNET = 'vaspdirectory.net',
+  MAINNETDEV = 'vaspdirectory.dev'
```

Comment:
I added `MAINNETDEV` in case we get a response where the `registered_directory` has the `.dev` domain.